### ISSUE TITLE
quote_command_options causing ffmpeg to fail

### ISSUE THREAD

**Issue by jessecurry** [thoughtbot/paperclip#240]
I'm in the process of setting up a site that allows the user to upload video content, thumbnails are created using ffmpeg.

Unfortunately, every time I uploaded a video the thumbnail process would fail. I managed to trace the problem to the command that was being generated. This issue was that the quote_command_options method was adding ' characters around the command that was being executed.

I managed to fix my problem by changing line 133 of paperclip.rb:
     option.split("'").map{|m| "'#{m}'" }.join("\'")
to
     option.split("'").map{|m| "#{m}" }.join("\'")

basically just removing the single quotes.

I'm very new to ruby and rails, is my change something that will cause problems elsewhere? What is the purpose of the single quotes?
